src/components/Home.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import getToken from '../lib/tokens.js';
+import { useHistory } from 'react-router-dom';
+
+const Home = () => {
+  const history = useHistory();
+
+  const generateToken = () => {
+    const token = getToken();
+    localStorage.setItem('token', token);
+    history.push('/ListView');
+  };
+
+  return <button onClick={generateToken}>Create a new list</button>;
+};
+
+export default Home;
```

Comment:
I found this component really nifty and clean! It works very nicely with our router.